docs/section18/Rule_18-1

```diff
@@ -0,0 +1,72 @@
+**Rule Assertion:** B-RMR = expected value  
+**Appendix G Section:** Section 18 HVAC_SystemZoneAssignment  
+**Appendix G Section Reference:** Table G3.1.1.3
+
+**Applicability:** All required data elements exist for B_RMR  
+**Applicability Checks:**  
+
+1. Buildings in CZ 0-3A
+2. predominant HVAC BAT = public assembly
+3. building area < 120,0000 ft2
+4. the zone does not meet one if the G3.1.1.1 exceptions
+
+**Manual Check:** Yes  
+**Evaluation Context:** Evaluate whether each Zone is served by the correct HVAC system type  
+**Data Lookup:**   
+**Function Call:** 
+
+1. zone_meets_any_G3_1_1_exception(P-RMR,B-RMR,zone)
+2. get_baseline_system_types()
+3. get_HVAC_building_area_types_and_zones
+3. get_baseline_system_types
+
+**Applicability Checks:**
+
+1. Check if the climate zone is CZ0-3A: `ASHRAE229.weather.climate_zone in ["CZ0A","CZ0B","CZ1A","CZ1B","CZ2A","CZ2B","CZ3A"]`
+	- set weather_applicable = TRUE: `weather_applicable = TRUE`
+	
+2. get the list of HVAC systems and zones: `list_of_systems_and_zones = get_HVAC_building_area_types_and_zones(B-RMR)`
+	- find the predominant HVAC BAT by first sorting the list by area: `sorted_list = sorted(list_of_systems_and_zones.items(), key=lambda x: x[1]["AREA"], reverse = True)`
+	- the predominant building area type will be the first item in the sorted list: `if sorted_list[0][0] == "PUBLIC_ASSEMBLY":`
+		- set area_type_applicable to true: `area_type_applicable = TRUE`
+		
+3. find the building area and check whether it's less than 120,000 ft2.  Start by creating an area variable: `building_area = 0`
+	- We can use the list_of_systems_and_zones to get the total building floor area.  Start by looping through the list: `for x in list_of_systems_and_zones:`
+		- x will be the various HVAC BAT.  list_of_systems_and_zones[x]["AREA"] will give the total area of this BAT: `building_area = building_area + list_of_systems_and_zones[x]["AREA"]
+	- check if building_area < 120000: `if building_area < 120000:`
+		- set area_applicable to true: `area_applicable = TRUE` 
+
+
+4. make a list of eligible zones: `eligible_zones = []`
+- loop through building segments: `for segment in RMR.building_segments:`
+	- loop through zones in the segments: `for zone in segment.zones:`
+		- check whether the zone meets one of the exceptions: `if(zone_meets_any_G3_1_1_exception(P-RMR,B-RMR,zone) == FALSE:`
+			- zone should take the predominant system type: `zone_applicable = TRUE`
+			- add zone to list: `eligible_zones.append zone`
+
+- if the 3 first checks are TRUE, the rule applies else, rule is not applicable to B-RMR: `else: RULE_NOT_APPLICABLE`
+
+## Rule Logic:  
+- call get_baseline_system_types and save a list of baseline HVAC system types: `hvac_system_types = get_baseline_system_types(B-RMR)`
+- for each zone that passed the zone applicability test above: `for zone in eligible_zones:`
```

Comment:
The appicability checks shoudl be expressed in pseudo code withing the rule logic.

Reply:
We already have list of HVAC systems associated with each zone - [get_list_hvac_systems_associated_with_zone.md](https://github.com/pnnl/ruleset-checking-tool/blob/develop/docs/ruleset_functions/get_list_hvac_systems_associated_with_zone.md)
So I think you just need to call this function for the "eligible_zone" and confirm that the zone has only one HVAC system and that system is of type 4.